MetaMask/metamask-extension
# UX: Multichain: Add separator under 'View on Explorer'

## **Description**
This PR adds a separator to the Multichain global menu under View on Explorer

## **Manual testing steps**

1.  Open the Global Menu
2. See the separator between View on Explorer and its next menu item

## **Screenshots/Recordings**

<img width="290" alt="SCR-20231005-txgc" src="https://github.com/MetaMask/metamask-extension/assets/46655/5e8dbbd5-45dc-4eda-999a-117580569190">


## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/pull/21220

This issue also references the position of the Account Picker but that is addressed in https://github.com/MetaMask/metamask-extension/pull/21220

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
